### DISCUSSION TITLE
Long index for native arrays.

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [beehive-lab/TornadoVM#499] [Q&A] [answered]
Good day.
Is there anything that prevents the addition of support for long indexes for TornadoNativeArray-s? 
Would you be against or pro for such PR?

By long-based indexes, I mean get/set methods that accept long instead of int.

**Comment by jjfumero** [comment, marked as answer]
Hi @andrii0lomakin, there is nothing that prevents this. In fact, this will be a great addition. Would you like to work on this PR?

**Reply by andrii0lomakin** [reply]
Hi. 
Sure, I suppose that will be a good starting point.

I am on vacation now, but I will do it once I return.
